platforms/wechat/wrapper/engine/VideoPlayer.js

```diff
@@ -300,6 +300,9 @@
         this._h = node._contentSize.height;
 
         let camera = cc.Camera.findCamera(node);
+        if (!camera) {
+            return;
+        }
```

Comment:
Should validate camera at the beginning, just after line 282

Reply:
moved